packages/phrases/src/locales/en/errors/connector.ts

```diff
@@ -35,6 +35,9 @@ const connector = {
     'You can not have multiple social connectors that have same target and platform.',
   cannot_overwrite_metadata_for_non_standard_connector:
     "This connector's 'metadata' cannot be overwritten.",
+  saml_only_idp_initiated_auth: 'Only the SAML connector can set up IdP initiated authentication.',
+  invalid_application_type:
```

Comment:
can use another key with more specific name? It seems the key in this error message represents a broader scenario than its actual usage.